idtf2u3d/idtf2u3d.m

```diff
@@ -114,4 +114,6 @@
 
 function [fname] = full_fname_with_extension(fname, extension)
 fname = check_file_extension(fname, extension);
-fname  = fullfile(cd, fname);
+if(isempty(fileparts(fname)))

```

Comment:
Could you please give an example that explains why the `isempty` function results in handling full paths here? It seems that the function `fileparts` is the one documented at: https://www.mathworks.com/help/matlab/ref/fileparts.html .